lib/pbench/cli/agent/commands/results/__init__.py

```diff
@@ -157,13 +191,13 @@ def execute(self):
     help="pbench server authentication token (will prompt if unspecified)",
 )
 @click.option(
-    "--prefix", help="",
+    "--prefix", default=None, help="Pbench prefix (Obsolete)",
 )
 @click.option(
-    "--xz-single-threaded", help="",
+    "--xz-single-threaded", help="Single threaded operation",
```

Comment:
Will change the help message couldn't find appropriate msg at present

Reply:
@portante : Do we really even need to expose compressor options to the user? Has anyone ever used this???

Reply:
Good question. This was added a while back so that folks can reduce the CPU consumption of the compression phase to prevent it from taking over most of the CPU on the controller.

Reply:
Yes, we need to take this into account when we pull `MakeResultTb` out of the mothballs.